src/util.cc

```diff
@@ -100,11 +102,26 @@ static void MakeUtf8String(Isolate* isolate,
                            MaybeStackBuffer<T>* target) {
   Local<String> string;
   if (!value->ToString(isolate->GetCurrentContext()).ToLocal(&string)) return;
+  String::ValueView value_view(isolate, string);
+
+  if (value_view.is_one_byte() && simdutf::validate_ascii(reinterpret_cast<const char*>(value_view.data8()),value_view.length()) ) {
```

Comment:
Can we also have a comment explaining the if-reason?

Reply:
Why was this marked as "resolved"?

Reply:
Because I removed the `validate_ascii` part, and the `is_one_byte()` is pretty self explanatory, isn't it?